### PR TITLE
added html and css files

### PR DESCRIPTION
**Pull request by shakerAbuDrais**
added html and css files to the project

### CODE REVIEW

**Comment by shakerAbuDrais** [comment]
Great job so far, you are almost done. 

Required*
here are a few comments that might help improve your code: 

I think that you have forgotten to use linters, it is easy to use them just follow[ this link ](https://github.com/microverseinc/linters-config/tree/master/html-css)to install linters.

It would look more professional if you commit differently instead of saying "added HTML and CSS to the project" say "Add HTML home page and its styling".

next comments will be inline comments please refer to the code and check them.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+        <ul id="menu">
+          <a class="nav-link" href="#features"> features</a>
+          <a class="nav-link" href="#howitworks"> how it works</a>
+          <a class="nav-link" href="#pricing"> pricing</a>
+        </ul>
+      </nav>
+    </header>
+    <main id="main">
+      <h2>this is the home page of the store</h2>
+      <div class="">
```
Comment (shakerAbuDrais) [inline index.html:24]: please do not use empty classes

File: index.html [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+        <form id="form" action="https://www.freecodecamp.com/email-submit">
+          <input id="email" type="email" name="email" placeholder="please enter your email">
+          <input id="submit" type="submit" name="" value="get started">
+        </form>
+      </div>
+      <section id="features">
+        <div class="desc">
```
Comment (shakerAbuDrais) [inline index.html:31]: please, use meaningful class names as you are coding with someone else.

File: index.html (hunk 1 above)
Comment (Abenezer-Tilahun) [reply index.html:24]: - [x] Please try to  includes an explanation, a "why", for each requested change.
Example "Please Why don't you Apply best practices in your HTML code , leaving any empty classes in your HTML code makes your code-base not clear "

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @shakerAbuDrais , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- [x] Good Job for phrased the comments politely, as suggestions instead of orders.
- [x] Good Job for using git-hub inline comments to leave review comments.
- [x] Good job for providing Summarize for Requested changes 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**